gprofiler/profilers/python.py

```diff
@@ -340,6 +342,16 @@ def _should_skip_process(self, process: Process) -> bool:
             action="store_true",
             help="Enable PyPerf in verbose mode (max verbosity)",
         ),
+        ProfilerArgument(
+            name="--python-pyspy-processes",
+            dest="python_pyspy_processes",
+            nargs="+",
```

Comment:
From argparse docs:
```
'+'. Just like '*', all command-line args present are gathered into a list. Additionally, an error message will be generated if there wasn’t at least one command-line argument present. For example:
```

If I understand correctly, doesn't it require one value?

Reply:
It requires one only if the specific argument appears
that is the desired case here I guess (we don't want to support when people specify `--python-pyspy-processes` without PIDs)

Reply:
I think the nicer way is to name the arg singular and use `action="extend"`, so it's provided this way: `--python-pyspy-process xxx --python-pyspy-process yyy`

Reply:
Feel free to resolve after finishing with @Jongy's comment

Reply:
I preferred the old way

Reply:
@Jongy I resolve this comment because you approve this PR